prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1673,17 +1673,23 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 	 * Resend handshake packet for any peers where the first
 	 * handshake send failed.
 	 */
-	dlist_foreach_container_safe(&ep->peer_queued_list,
+	dlist_foreach_container_safe(&ep->handshake_queued_peer_list,
 				     struct rdm_peer, peer,
-				     queued_entry, tmp) {
+				     handshake_queued_entry, tmp) {
+
```

Comment:
it looks like this isn't new, but it doesn't seem like there should be a blank newline here?

Reply:
removed the new line in new revision